packages/javascript/bh-shared-ui/src/components/AppIcon/Icons/CaretDown.tsx

```diff
@@ -19,14 +19,7 @@ import { BasePath, BaseSVG, BaseSVGProps } from './utils';
 
 export const CaretDown: React.FC<BaseSVGProps> = (props) => {
     return (
-        <BaseSVG
-            name='caret-down'
-            width='10'
-            height='6'
```

Comment:
see AppIcon/Index for more context. Essentially, these width/height properties weren't getting used previously and only existed from copy/pasting from figma files. With changes to the `BaseSVG`, we can now size icons from their callsites with height/width which is useful for non-square icons.